use failure::Error;

pub fn run(input: &str) -> Result<String, Error> {
    Ok(String::from("foo"))
}
